tools/wptrunner/wptrunner/environment.py

```diff
@@ -294,3 +295,27 @@ def test_servers(self):
                         s.close()
 
         return failed, pending
+
+
+def wait_for_service(addr, timeout=60):
+    """Waits until network service given as a tuple of (host, port) becomes
+    available or the `timeout` duration is reached, at which point
+    ``socket.error`` is raised."""
+    print("Trying to connect to %s" % (addr,))
```

Comment:
This `print` is added in this PR. It will print something like "Trying to connect to ('localhost', 8080)" because `addr` is a tuple.

I'm pretty sure this should use `logger` to not log unconditionally, and with that maybe fix the format with `f"Trying to connect to {addr[0]}:{addr[1]}"`